libzmq-sys/src/bindings.rs

```diff
@@ -210,6 +210,9 @@ pub const ZMQ_CURRENT_EVENT_VERSION: u32 = 1;
 pub const ZMQ_CURRENT_EVENT_VERSION_DRAFT: u32 = 2;
 pub const ZMQ_EVENT_ALL_V1: u32 = 65535;
 pub const ZMQ_EVENT_ALL_V2: u32 = 131071;
+pub type __uint8_t = ::std::os::raw::c_uchar;
```

Comment:
What's up with that?

Reply:
These were generated when doing a standard `cargo build`. I assume it's due to a bindgen update.